api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -795,6 +795,9 @@ def define_liquid(
         :param str name: A human-readable name for the liquid.
         :param str description: An optional description of the liquid.
         :param str display_color: An optional hex color code, with hash included, to represent the specified liquid. Standard three-value, four-value, six-value, and eight-value syntax are all acceptable.
+
+        :return: a :py:class:`opentrons.types.Liquid` representing the specified
+            liquid.
```

Comment:
```suggestion
        :return: A :py:class:`~opentrons.protocol_api.Liquid` object representing the specified liquid.
```
fix and shorten cross-reference